core/src/test/java/io/grpc/internal/ConfigSelectingClientCallTest.java

```diff
@@ -121,6 +122,7 @@ public void selectionErrorPropagatedToListener() {
     InternalConfigSelector configSelector = new InternalConfigSelector() {
       @Override
       public Result selectConfig(PickSubchannelArgs args) {
+        // This status code is considered inappropriate to propagate from the control plane...
         return Result.forError(Status.FAILED_PRECONDITION);
```

Comment:
Could you make a copy of this test, and have one version use an appropriate status code? Because that's the common case that should be tested.